evalme/video/video.py

```diff
@@ -9,6 +9,64 @@
 class VideoEvalItem(EvalItem):
     SHAPE_KEY = 'videorectangle'
 
+    def rectangle_simple(self, pred, per_label=False, **kwargs):
+        """
+        Simple score for video frames
+
+        :param pred: Predicted VideoEvalItem
+        :param per_label: per label calculation flag
+        :return: float or dict of floats
+        """
+        control_weights = kwargs.get('control_weights', {})
+        control_weights_overall = control_weights.get('box', {}).get('overall', 0)
+        labels_weights = control_weights.get('videoLabels', {}).get('labels', {})
+        labels_weights_overall = control_weights.get('videoLabels', {}).get('overall', 1.0)
+        # prepare results vars for per_label
+        if per_label:
+            results = defaultdict(float)
+            results_count = defaultdict(int)
+        else:
+            results = 0
+            results_count = 0
+        # extract ALL frames from raw data
+        gt_frames_results = self.get_frames()
+        pred_frames_results = pred.get_frames()
+        # check each predicted frame set score with ground truth frame set
+        for pred_frame in pred_frames_results:
+            if per_label:
+                max_value = {}
+            else:
+                max_value = 0.0
+            for gt_frame in gt_frames_results:
+                labels_gt = gt_frame['value'].get('labels', [])
+                labels_pred = pred_frame['value'].get('labels', [])
+                res_labels = int(labels_gt == labels_pred) * labels_weights[labels_gt[0]]
```

Comment:
I noticed in the above code you allow labels_gt to default to an empty list `labels_gt = gt_frame['value'].get('labels', [])`.  However if it is ever empty, `labels_gt[0]` here will throw an IndexError.  This should be accounted for.